streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBKeyValueStoreSupplier.java

```diff
@@ -29,7 +29,7 @@
  * @param <V> the type of values
  * @see org.apache.kafka.streams.state.Stores#create(String)
  */
-
+@Deprecated
```

Comment:
Generally, one should never have a @Deprecated annotation without a @deprecated in the javadoc with an explanation of when it was deprecated, what is the replacement and typically a note saying that it will be removed in a future major version.

Reply:
This is an internal class, so I thought it would be ok with JavaDoc ?

Reply:
Even if it's internal, it's useful to document the context? See https://docs.oracle.com/javase/tutorial/java/annotations/predefined.html for the recommendation that both should be used.